src/NUnitFramework/nunitlite/TextRunner.cs

```diff
@@ -318,21 +318,21 @@ public void ReportResults(ITestResult result)
             _textUI.DisplaySummaryReport(Summary);
         }
 
-        private int ExploreTests()
+        private int ExploreTests(ITestFilter filter)
         {
 #if !PORTABLE
             ITest testNode = _runner.LoadedTest;
```

Comment:
Ok - I think that I misunderstood here...

The runner in nunitlite comes from:
`private ITestAssemblyRunner _runner;`

And there is no `Explore` method on that interface. Hence I updated the current code...

If I look at the `FrameworkController` code for the other `ExploreTests` implementation I can see why putting it into the interface is preferable. I had missed this last time I looked at it.

I'll look at implementing the `ExploreTests` onto that interface.

On that - is there any objection/reason the code should not re-read the assembly to filter? i.e. if I re-use the builders to re-build the tests based on a filter set - or is it preferable to just modify the Test list?